RecoLocalTracker/SiPixelClusterizer/python/SiPixelClusterizerPreSplitting_cfi.py

```diff
@@ -1,7 +1,9 @@
-
 import FWCore.ParameterSet.Config as cms
 
-#
 from CondTools.SiPixel.SiPixelGainCalibrationService_cfi import *
 from RecoLocalTracker.SiPixelClusterizer.SiPixelClusterizer_cfi import siPixelClusters as _siPixelClusters
 siPixelClustersPreSplitting = _siPixelClusters.clone()
+
+# *only for the cms-patatrack repository*
+# ensure reproducibility for CPU <--> GPU comparisons
+siPixelClustersPreSplitting.payloadType = 'HLT'
```

Comment:
I'd expect this to get automatically propagated from `SiPixelClusterizer_cfi`.

Reply:
Yes, indeed.